### DISCUSSION TITLE
Can we please making beet missing -a more useful?

### DISCUSSION THREAD

**Discussion by Xavier-Waters** [beetbox/beets#4367] [Ideas]
Beet missing -a currently prints out all missing albums, singles, compilations, eps, etc. Can we make it print out solely albums and eps? I'm sure this would be much more useful as I would imagine most people do not want every single release ever from every artist. Additionally the documentation claims beet missing -a lists all missing albums where it definitely does not.

**Comment by wisp3rwind** [comment]
Seems like a reasonable feature (as in, I'd be happy to review a pull request). Maybe, it would be good to design the CLI such that the old and new (full and album-only) output is possible. A first idea would be to add a `--kind` option, which can take a comma-separated list of `ep,album,single,...`.

**Reply by sampsyo** [reply]
Indeed! An even more general (and perhaps simpler to implement) option would be to have the option to filter based on a query. A reasonable filter could look like `albumtype:album | albumtype:ep`.

**Reply by Xavier-Waters** [reply]
Thanks, this seems like a great solution.

**Reply by Xavier-Waters** [reply]
Hey, is this being worked on?
